api/store.ts

```diff
@@ -2,10 +2,17 @@ import * as path from 'path';
 import * as nedb from 'nedb';
 import * as Q from 'q';
 
-export class Store {
-    constructor() {
+export interface IStore {
```

Comment:
program to an interface instead of a concrete implementation to provide flexibility